cpp/data/system_updates/8.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+cd /usr/local/ub_tools/cpp
+make convert_rss_db_tables
+./convert_rss_db_tables
```

Comment:
import_rss_aggregator_conf must be called first, or we won't have any feeds in the db which we can insert the items for!